docs/howtos/runtime_sessions.rst

```diff
@@ -0,0 +1,70 @@
+Use Experiments with Runtime sessions
+=====================================
+
+Problem
+-------
+
+You want to run experiments in a `Runtime session
+<https://qiskit.org/ecosystem/ibm-runtime/sessions.html>`_ so that jobs can run in close temporal proximity.
```

Comment:
Would it be better for the examples to show two experiments? It's not clear from them that they require multiple jobs that would benefit from sessions. Or maybe use some other way to point out that this helps group multiple jobs together, like setting the `max_circuits` experiment option?

Reply:
Good idea, I update the example to set `max_circuits`.